api/v1alpha1/dnsrecord_types.go

```diff
@@ -162,6 +162,7 @@ type DNSRecordStatus struct {
 //+kubebuilder:object:root=true
 //+kubebuilder:subresource:status
 //+kubebuilder:printcolumn:name="Ready",type="string",JSONPath=".status.conditions[?(@.type==\"Ready\")].status",description="DNSRecord ready."
+//+kubebuilder:printcolumn:name="Healthy",type="string",JSONPath=".status.conditions[?(@.type==\"Healthy\")].status",description="DNSRecord healthy.",priority=2
```

Comment:
is there a way to make this display something like "unknown" rather than nothing at all, when the field is missing?